src/states/adult.rs

```diff
@@ -195,7 +195,7 @@ impl Adult {
     // Can also just be a single target once node-ageing makes Offline votes Opaque which should
     // remove invalid test failures for unaccumulated parsec::Remove blocks.
     fn send_parsec_poke(&mut self) {
-        let version = *self.gen_pfx_info.first_info.version();
+        let version = self.gen_pfx_info.parsec_version;
```

Comment:
can you search in code if there are other places using `gen_pfx_info` or the struct that we did not update?

Reply:
had searches with `gen_pfx_info`, `GenesisPrxInfo`, `ParsecPoke`, `first_info.version()`, `ParsecRequest` & `ParsecResponse`
all got updated.